x/poe/contract/tgrade_valset_integration_test.go

```diff
@@ -286,5 +286,33 @@ func TestJailUnjail(t *testing.T) {
 			assert.Empty(t, res.Validator.JailedUntil)
 		})
 	}
+}
 
+func TestIterateActiveValidators(t *testing.T) {
+	specs := map[string]struct {
+		valCount int
+	}{
+		"list single pages": {
+			valCount: 30,
+		},
+		"list two pages": {
+			valCount: 31,
+		},
+		"list all actives": {
+			valCount: 100,
+		},
+	}
+	for name, spec := range specs {
+		t.Run(name, func(t *testing.T) {
```

Comment:
This is not a true test, unless you assert a limit in `IterateActiveValidators` so we know the page size is really 30 (maybe the default is 200?).

Reply:
Correct! I verified though that it returns 30 as default but there is no guarantee for the future. I went forth and back with adding a limit to the query or not. Later, I removed it to rely on the contract developers work and design choice.
I give it another thought now. Thanks for finding this!